File: 1832-check-if-the-sentence-is-pangram/1832-check-if-the-sentence-is-pangram.cpp
class Solution
{
public:
    bool checkIfPangram(string sentence)
    {
        char c = 'a';
        int cnt = 0;
        while (c <= 'z')
        {
            for (int i = 0; i < sentence.length(); i++)
            {
                if (sentence[i] == c)
                {
                    cnt++;
                    break;
                }
            }
            c++;
        }
        if (cnt == 26)
            return 1;
        else
            return 0;
    }
};

File: 1832-check-if-the-sentence-is-pangram/1832-check-if-the-sentence-is-pangram.java
class Solution {
    public boolean checkIfPangram(String sentence) {
        int freq[]=new int[26];
        for(int i=0;i<sentence.length();i++){
            int ind=sentence.charAt(i)-'a';
            freq[ind]++;
        }
        for(int i=0;i<26;i++){
            if(freq[i]==0)return false;
        }

        return true;
    }
}